core/src/test/java/uk/gov/gchq/koryphe/impl/function/StringJoinTest.java

```diff
@@ -48,7 +49,10 @@ public void shouldHandleNullInput() {
     public void shouldHandleNullDelimiter() {
         // Given
         final StringJoin<String> function = new StringJoin<>(null);
-        final Set<String> input = Sets.newHashSet("a", "b", "c");
+        final Set<String> input = new LinkedHashSet<>();
```

Comment:
Maybe an ArrayList could be used instead to keep this all on one line?

Reply:
Oh. right. I assumed you wanted a set. but using a ```List``` makes sense as I believe it is deterministic and should solve the flakiness in the code!